src/components/grid-aware/Footer/SubscriptionBlock/SubscriptionBlock.js

```diff
@@ -0,0 +1,35 @@
+import PropTypes from "prop-types";
+import React from "react";
+import Button from "../../../inline/Button";
+import InputText from "../../../inline/InputText";
+
+import s from "./SubscriptionBlock.module.css";
+
+const SubscriptionBlock = ({ placeholderText, type }) => {
+  return (
+    <div className={s.bleedWrapper}>
+      <div className={s.subscriptionBlock}>
+        <div className={s.subscribeContainer}>
+          <div className={s.title}>Subscribe to get updates</div>
+          <form className={s.formInput}>
```

Comment:
`formInput` is a pretty confusing class name for the `form`, since it makes it sound like it's for an `<input>` element. Can you give this a different name?

Reply:
just changed to .form for simplicity